lib/src/select/Select.tsx

```diff
@@ -151,8 +147,8 @@ const DxcSelect = React.forwardRef<RefType, SelectPropsType>(
     const [visualFocusIndex, changeVisualFocusIndex] = useState(-1);
     const [isOpen, changeIsOpen] = useState(false);
 
-    const selectRef = useRef(null);
-    const selectSearchInputRef = useRef(null);
+    const selectRef = useRef<HTMLInputElement | null>(null);
```

Comment:
The select ref is on a div element (<Select> styled-component). We should type this one as an `HTMLDivElement`.

Reply:
This affects the `useWidth` parameter, it should also be an `HTMLDivElement` element.